pkl-executor/pkl-executor.gradle.kts

```diff
@@ -52,10 +52,15 @@ sourceSets {
   }
 }
 
+val copyHistoricalDistribution by tasks.registering(Copy::class) {
```

Comment:
Should match the configuration name (which can of course be renamed). What's needed is a distribution that doesn't have ExecutorSpiOptions2. That's why I used the overly specific name `025`.

Reply:
I am leaning towards "historical" because when there's the inevitable ExecutorSpiOptions3, maybe we will need to download another distribution that has `ExecutorSpiOptions2` but is missing `ExecutorSpiOptions3`.

Reply:
Oh I see. You want to put all distributions into a single configuration.

Reply:
Yup, exactly!

> By the way, would be easy to create symlinks instead of copying (iterate over the configuration in doLast and use Files.createSymbolicLink).

Switched to making symlinks